nilearn/tests/test_signal.py

```diff
@@ -358,12 +379,13 @@ def test_clean_confounds():
 
     # Test without standardizing that constant parts of confounds are
     # accounted for
-    np.testing.assert_almost_equal(nisignal.clean(np.ones((20, 2)),
-                                                  standardize=False,
-                                                  confounds=np.ones(20),
-                                                  detrend=False,
-                                                  ).mean(),
-                                   np.zeros((20, 2)))
+    for normalize in [None, 'psc']:
```

Comment:
from my side nothing. I was thinking about adding an example to explain the different normalization options, but as part of a more general demo for different preprocessing steps (discussion #967)